src/ui/public/kuery/suggestions/value.js

```diff
@@ -0,0 +1,48 @@
+import chrome from 'ui/chrome';
+import { escapeQuotes } from './escape_kql';
+import { escape } from 'lodash';
+
+const baseUrl = chrome.addBasePath('/api/kibana/suggestions/values');
+const type = 'value';
+
+function getDescription({ fieldName, value }) {
+  return `
+    <p>Find results where <span class="suggestionItem__callout">${escape(fieldName)}</span>
+    is <span class="suggestionItem__callout">${escape(value)}</span></p>
+  `;
+}
+
+export function getSuggestionsProvider({ $http, config, indexPattern }) {
```

Comment:
This seems to be the only suggestion provider using `$http` and `config`. 

I don't think you need to use `$http` at all, since you're [wrapping the response in a native promise upstream](https://github.com/lukasolson/kibana/blob/f2a0832e13ceaf5c2292e96abe3de2961bb4fc21/src/ui/public/kuery/suggestions/index.js#L27) anyway, and then [later using $scope.$apply](https://github.com/elastic/kibana/pull/15945/files/f2a0832e13ceaf5c2292e96abe3de2961bb4fc21#diff-a63e1054ef0dce774fee5c3a7c450184R64). `Fetch` or some wrapper would give you the same functionality, and remove the Angular dependency here.

And the only thing you're using the `config` service for is to check the value of `filterEditor:suggestValues`. Seems like it would be better to get this (and any other config value you might need) upstream and pass down a config object of some kind. It'll make this code easier to decouple from Angular in the future too, no? This one may be a bit moot since you still need to `indexPattern` and `persistedLog` services anyway...